xml/ay_bigfile.xml

```diff
@@ -1029,6 +1029,37 @@ exit 0
       <row>
        <entry>
         <para>
+         <literal>self_update</literal>
+        </para>
+       </entry>
+       <entry>
+        <para>
+         This option allows to enable/disable the self-update.
+         Check out the &deploy; to find further information about this feature.
+         </para>
+        <screen>&lt;self_update config:type="boolean"&gt;
+  false
+&lt;self_update/&gt;</screen>
+        <para>
+         Alternatively, you can specify the boot parameter
+         <literal>self_update</literal> on the Kernel command line.
+        </para>
+        <important>
+         <title>Disabling the self-update</title>
+         <para>
+          Please, note that this feature will be skipped if any option disables
+          it: setting this option to <literal>false</literal> or using the
+          boot parameter <literal>self_update=0</literal>.
+         </para>
+        </important>
```

Comment:
If you document true/false and 0/1 as suggested above, the complete <important> section can be removed.